src/Components/Display/Menu/UpMenu.tsx

```diff
@@ -42,11 +43,14 @@ class UpMenu extends React.Component<UpMenuProps & WithThemeProps, UpMenuState>{
     constructor(p, c) {
         super(p, c);
         this.state = {
-            minified: this.props.minified,
+            minified: this.props.minified || false,
         };
     }
 
     toggleMinification = () => {
+        if(this.props.minifyHandler){
+            this.props.minifyHandler(!this.state.minified);
+        }
         this.setState({ minified: !this.state.minified });
```

Comment:
Mettre dans le callback : 
this.setState({ minified: !this.state.minified }, () => {
     if(this.props.minifyHandler){
            this.props.minifyHandler(!this.state.minified);
     }
});